### ISSUE TITLE
Can i use this framework for my commercial WordPress project?

### ISSUE THREAD

**Issue by ulziibat-n** [CherryFramework/cherry-framework#182]
Hello.
I am WordPress developer.
I want to use interface builder module to my commercial project.
I seen Cherry Framework is 100% GPL. But i need to ask it from framework owner. I am planned to use it for my house premium theme and plugins.

Thanks.

**Comment by Swizzler121** [comment]
@dimative Yeah, check out the license, it's GNU General Public License v3.0. If you click on the license in github it does a nice little breakdown of where you can use it and what you can do with it, and one of the things it says you can use it in is commercial use.
